test/functional/ex_cmds/mksession_spec.lua

```diff
@@ -0,0 +1,53 @@
+-- Test generation of session files
+
+local lfs = require('lfs')
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local get_pathsep = helpers.get_pathsep
+local eq = helpers.eq
+local funcs = helpers.funcs
+
+
+describe('A session file', function()
+  -- The file name use for each session
+  local session_file = 'Xtest-functional-ex_cmds-mksession.vim'
+  -- Directory for testing tab-local working directories
+  local tab_dir = 'Xtest-functional-ex_cmds-mksession_spec.tab'
+
+  before_each(function()
+    clear()
+    lfs.mkdir(tab_dir)
+  end)
+
+  -- Each test should start with a clean session file
```

Comment:
Comment would be logical if cleanup was in `before_each()`, not here and with a single test only. Actually cleanup has two purposes: first, have a clean state for the next tests or the next test suite runs. Second, do not make various files appear in `git status`: current working directory of busted is a root directory of the repository, `Xtest*` files are not ignored.

And other tests do not have such comments.